### ISSUE TITLE
Dexie.Syncable with existing DB

### ISSUE THREAD

**Issue by emilienh** [dfahlander/Dexie.js#946]
Hi and first of all thanks a lot for this great library. 

I am trying to implement Dexie.Syncable to download an existing MongoDB database that already has over 40k records. It wasn't built with incremental changes in mind. I was hoping to implement the  ISyncProtocol with an array of new changes stored in memory on the server, that would automatically purge changes after a few days, and any sync requested with an older revision would simply receive the entire DB again.

From the sample code and various implementations I've seen, it doesn't seem like the server ever sends actual records to the clients, but only incremental changes. Would my approach even work with this protocol? In order to perform the initial sync with the client, would my only choice be to create 40k changes and send them up every time a client subscribes for the first time? 

Thanks a lot in advance.

**Comment by dfahlander** [comment]
Yes, you'd need to send the all data that you want to be in sync on the client as CREATE changes.
